src/table/interfaces.tsx

```diff
@@ -280,6 +280,7 @@ export interface TableProps<T = any> extends BaseComponentProps {
    *                  (such as in a dashboard item container).
    * * `embedded` - Use this variant within a parent container (such as a modal, expandable
    *                section, container or split panel).
+   *                **Deprecated**, replaced by `borderless` and `container`.
```

Comment:
In flashbar docs, we render deprecation notes on the same line with the text

https://github.com/cloudscape-design/components/blob/589b1d20eab865a1ed76be799121d7e9f2e4419d/src/flashbar/interfaces.ts#L61-L62

I think this is needed to preserve list formatting. Can you check how it renders before the merge? Or just put on a single line for safety?